catalog/security/catalog-security-plugin/src/main/java/ddf/catalog/security/plugin/SecurityPlugin.java

```diff
@@ -56,7 +56,7 @@ public CreateRequest processPreCreate(CreateRequest input) throws StopProcessing
             input.getMetacards()
                     .forEach(metacard -> {
                         metacard.setAttribute(new AttributeImpl(Metacard.POINT_OF_CONTACT,
-                                SubjectUtils.getName((ddf.security.Subject) subjectName)));
+                                SubjectUtils.getEmailAddress((ddf.security.Subject) subjectName)));
```

Comment:
That's a good point on your first comment. The email address actually is `null` if the user is not logged in, in which case `point-of-contact` does not get set or persisted. This is exactly what we want, especially for the security exception that the second PR will implement since a guest will never get a security exception.